spec/system/home_page_spec.rb

```diff
@@ -0,0 +1,45 @@
+require "rails_helper"
+
+RSpec.feature "Home Page" do
+  scenario "User visits the claims homepage" do
+    given_i_am_on_the_claims_site
+    and_i_am_on_the_start_page
+    i_can_see_the_claims_service_name_in_the_header
+  end
+
+  scenario "User visits the placements homepage" do
+    given_i_am_on_the_placements_site
+    and_i_am_on_the_start_page
+    i_can_see_the_placements_service_name_in_the_header
+  end
+
+  private
+
+  def given_i_am_on_the_claims_site
+    allow_any_instance_of(ApplicationHelper).to receive(
+      :current_service
+    ).and_return(:claims)
+  end
+
+  def given_i_am_on_the_placements_site
+    allow_any_instance_of(ApplicationHelper).to receive(
+      :current_service
+    ).and_return(:placements)
+  end
```

Comment:
Can I suggest something like this instead?

```suggestion
  def given_i_am_on_the_claims_site
    Capybara.app_host = "http://#{ENV["CLAIMS_HOST"]}"
  end

  def given_i_am_on_the_placements_site
    Capybara.app_host = "http://#{ENV["PLACEMENTS_HOST"]}"
  end
```

And then reset the `app_host` to the default value of `nil` in the test teardown:

```ruby
after do
  Capybara.app_host = nil
end
```

[Inspired by StackOverflow](https://stackoverflow.com/a/7050306), this seems to work as expected and allows the tests to pass.

I think this provides a more truthful system test because it covers the route config, and removes the need to stub the `current_service` helper method.